cfme/scripting/link_config.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+
+import click
+from pathlib2 import Path
+
+
+@click.command()
+@click.argument('src', type=click.Path(exists=True, file_okay=False))
+@click.argument('dest', type=click.Path(exists=True, file_okay=False))
+def main(src, dest):
+    """
+    links configfiles from one folder to another
+
+    if links exists it verifies content
+    if files exist at the target side it errors
+
+    Args:
+      src: source folder
+      dest: target folder
+    """
```

Comment:
Closer, but still not conforming to the standard. Please move the short description to the first line and use 4 spaces for indenting of the args. Some punctuation and capital letters would be appreciated as well.

Sorry for being pedantic here but we need to get into consistent state so all people have good comments in sight all the time when working on the project :).

Reply:
sure, can we get a linter error, i missed the my misfit there